### ISSUE TITLE
Remove one (redundant) line from Chart/Axis.php setAxisOptionsProperties(); change default value of $crossBetween to 'midCat'

### ISSUE THREAD

**Issue by bridgeplayr** [PHPOffice/PhpSpreadsheet#2932]
This is:

- [x] a bug report
- [ ] a feature request
- [ ] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)

### What are the steps to reproduce?

Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) of code that exhibits the issue without relying on an external Excel file or a web server:

 
Also - while you're in there making changes, change default value of $crossBetween because a non-zero value of minimum or maximum will not take effect unless $crossBetween is either midCat or 'between'

WAS:     private $crossBetween = ''; // 'between' or 'midCat' might be better
IS:         private $crossBetween = 'midCat'; // 

`        

        $this->setAxisOption('major_tick_mark', $majorTmt);
        $this->setAxisOption('minor_tick_mark', $minorTmt); //<---
        $this->setAxisOption('minor_tick_mark', $minorTmt);
        $this->setAxisOption('minimum', $minimum);`

If this is an issue with reading a specific spreadsheet file, then it may be appropriate to provide a sample file that demonstrates the problem; but please keep it as small as possible, and sanitize any confidential information before uploading.

### What features do you think are causing the issue

- [ ] Reader
- [ ] Writer
- [ ] Styles
- [ ] Data Validations
- [ ] Formula Calulations
- [x] Charts
- [ ] AutoFilter
- [ ] Form Elements

### Which versions of PhpSpreadsheet and PHP are affected?
1.24  7.3

**Comment by oleibman** [comment]
Changing the default value for crossBetween to null string gave *much* better results with the sample files. It is likely that none of the improved files specified min or max; nevertheless, I doubt that I will want to change the default back. It could make sense for the writer to use a default of midCat if it is null and min or max is non-zero. Please make that case, preferably with test file(s) to demonstrate the difference; and also please make the case for midCat being better than between (or for either of them being better than just documenting this requirement and relying on the user to read the documentation and add the additional statement).

Did you mean to close #2929?

**Comment by bridgeplayr** [comment]
I will examine sample files for impact of 'crossBetween'.  

In my project, Excel refused to display my generated ss after I assigned 'min'/'max' axis values.  When I allowed Excel to repair it, the key change made by Excel was the assignment of value 'midCat' to crossBetween.  Therefore, I will add 'min'/'max' values to axes in selected samples to observe the impact.

I did not mean to close the issue.  I meant only to close the Comment.  I am largely baffled by Github terminology.  Is it more appropriate to create a Pull Request?  It would seem that a user submits an ISSUE when the user has no implementation/correction suggestions.  Is it then more appropriate to create a PR when the user has suggested implementation code?

My style is to change my version of the library code to suit my needs, then submit an ISSUE rather than the seemingly scary PR.  This scheme runs the risk of falling behind on released versions, which I understand.  I considered creating a fork for my work, but file management made me hesitate to do that.

**Comment by oleibman** [comment]
I took sample file 32readwriteScatterChart2.xlsx, set minimum and maximum, and saved to a new file. I then edited the xml, deleting the crossBetween statements, so that the xml now contains c:min and c:max without c:crossBetween. I had no trouble opening this file. So, whatever the problem you saw in your corrupt file, the absence of crossBetween was either not a problem at all or, at least, only part of the problem.
